torch/csrc/jit/operator.cpp

```diff
@@ -103,65 +96,70 @@ struct SchemaParser {
     parseType(arg);
     args.push_back(std::move(arg));
   }
-  at::Tensor parseSingleConstant(TypeKind kind) {
+  IValue parseSingleConstant(TypeKind kind) {
     switch(L.cur().kind) {
       case TK_TRUE:
         L.next();
-        return one();
+        return true;
       case TK_FALSE:
         L.next();
-        return zero();
-      case TK_FLOAT:
-        L.next();
-        return as_tensor(static_cast<int64_t>(at::kFloat));
+        return false;
       case TK_IDENT: {
         auto tok = L.next();
         auto text = tok.text();
-        if("cpu" == text) {
-          return as_tensor(static_cast<int64_t>(at::Device::Type::CPU));
+        if("float" == text) {
+          return static_cast<int64_t>(at::kFloat);
+        } else if("cpu" == text) {
+          return static_cast<int64_t>(at::Device::Type::CPU);
         } else if("strided" == text) {
-          return as_tensor(static_cast<int64_t>(at::kStrided));
+          return static_cast<int64_t>(at::kStrided);
         } else if("ElementwiseMean" == text) {
-          return as_tensor(static_cast<int64_t>(Reduction::ElementwiseMean));
+          return static_cast<int64_t>(Reduction::ElementwiseMean);
```

Comment:
Yes! In fact that place is `ivalue.h` now, which is what replaced tensor_as and as_tensor. `toScalar` and `IValue(Scalar)` are already examples of this, since they do not actually store an at::Scalar.